TidepoolOnboardingKitUI/View Controllers/OnboardingRootNavigationController.swift

```diff
@@ -61,9 +60,32 @@ class OnboardingRootViewController: UIHostingController<AnyView>, OnboardingView
                 self.completionDelegate?.completionNotifyingDidComplete(self)
             }
             .store(in: &cancellables)
+        onboardingViewModel.$sectionProgression
```

Comment:
I assume this logic will eventually determine which section is not completed and then display that section?

Reply:
Not exactly, this logic is just for determining whether to set the root view to the Welcome screens tab view or the Getting To Know Tidepool Loop screen. Once the latter screen is set as the root view, there are a number of buttons that pop sheets for each section from that point on. The next root view change is when onboarding is complete and the home screen is shown.

Reply:
[nit] Hm, that wasn't obvious...maybe consider adding a code comment similar to your comment above?